source/switch/theme.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "colors.h"
+#include "image.h"
+
+typedef struct {
+    color_t backgroundColor;
+    color_t textColor;
+    color_t textActiveColor;
+    color_t textActiveBGColor;
+    Image btnA;
+    Image btnB;
+    Image btnX;
+    Image btnY;
+} theme_t;
+
+typedef enum {
+    LIGHT,
+    DARK,
+} themeMode;
```

Comment:
please either start a type name with a capital letter or name or end it with _t. 
Also if you don't mind, change the enum values to camel case with a prefix(enumNameEnumValue, similar to the UIResult enum)